sympy/polys/orthopolys.py

```diff
@@ -50,7 +52,7 @@ def jacobi_poly(n, a, b, x=None, **args):
     else:
         poly = PurePoly.new(poly, Dummy('x'))
 
-    if not args.get('polys', False):
+    if polys == False:
```

Comment:
Comparison == False seems unnecessary. I would expect `if not polys` or `if polys` with the two return statements switched, or a single return statement: `return poly if polys else poly.as_expr()`. I would probably use the latter one.